package.json

```diff
@@ -5,7 +5,7 @@
   "homepage": "./",
   "dependencies": {
     "@excalidraw/excalidraw": "npm:aakansha-excalidraw@0.2.1-cdn",
```

Comment:
Would you also try updating the main excalidraw package? I think the cdn asset is already supported.